packages/contractkit/src/wrappers/Validators.ts

```diff
@@ -343,6 +354,35 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     this.contract.methods.affiliate
   )
 
+  async getStatus(
```

Comment:
As written, this method seems a bit bespoke to be included in `contractkit`. It may be better to just leave it in the CLI code and extract it to `contractkit` when and if another user appears.

Reply:
Done